src/coreclr/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt

```diff
@@ -113,6 +113,12 @@ instructionset     ,X86   ,                     ,VectorT128         ,39 ,VectorT
 instructionset     ,X86   ,                     ,VectorT256         ,40 ,VectorT256            ,vectort256
 instructionset     ,X86   ,                     ,VectorT512         ,41 ,VectorT512            ,vectort512
 
+
+instructionset     ,X86   ,AvxVnniInt8          ,                   ,84 ,AVXVNNIINT            ,avxvnniint
+instructionset     ,X86   ,AvxVnniInt8_V512     ,                   ,85 ,AVXVNNIINT_V512       ,avxvnniint_v512
+instructionset     ,X86   ,AvxVnniInt16         ,                   ,86 ,AVXVNNIINT            ,avxvnniint
+instructionset     ,X86   ,AvxVnniInt16_V512    ,                   ,87 ,AVXVNNIINT_V512       ,avxvnniint_v512
```

Comment:
These can be `[60, 63]`. I intentionally left the gap since you had already used those in this PR.